### PR TITLE
tests: fix err check

### PR DESCRIPTION
**Pull request by Tjianke**
### What problem does this PR solve?

Issue Number: #22979  

Problem Summary:

### What is changed and how it works?

What's Changed: added err check

### Check List 

Tests 

- Unit test
- Integration test

### Release note 

- No release note

### CODE REVIEW

File: tests/graceshutdown/graceshutdown_test.go [hunk 1]
```diff
@@ -96,7 +96,8 @@ func (s *TestGracefulShutdownSuite) connectTiDB(port int) (db *sql.DB, err error
 		}
 		log.Warnf("ping addr %v failed, retry count %d err %v", addr, i, err)
 
-		db.Close()
+	err = db.Close()
+c.Assert(err, IsNil)
```
Comment (zimulala) [inline tests/graceshutdown/graceshutdown_test.go:100]: Please format this code.

Comment (Tjianke) [reply tests/graceshutdown/graceshutdown_test.go]: Done.

**Pull request merged**